src/chapters/setup-intro-data-types/more-data-types.rst

```diff
@@ -66,72 +115,22 @@ address* of the object, which we visualize as an arrow pointing from the
 variable box to where the data is stored. Instead of holding the actual ``Cat``
 data, ``myCat`` stores *directions* for finding the data in memory.
 
-When we assign a value to a reference type, as in ``Cat sameCat = myCat``, we
-do NOT create a second copy of the object or its data. Instead, we make a
-second arrow pointing to the same memory location.
+When we try to assign ``myCat`` to another variable, as in
```

Comment:
I'd keep the original language here. using 'try' connotes to me that there will be some obstacle to assigning it

Reply:
Fixed.  Thanks.